tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -144,5 +175,192 @@ function clusterRebalanceSuite() {
   };
 }
 
+function clusterRebalanceOtherOptionsSuite() {
+  return {
+
+    setUpAll: function () {
+      prevDB = db._name();
+      db._createDatabase(database);
+      db._useDatabase(database);
+      for (let i = 0; i < 3; ++i) {
+        db._create("col" + i, {numberOfShards: 2, replicationFactor: 2});
+      }
+      for (let i = 0; i < 3; ++i) {
+        for (let j = 0; j < 1000; ++j) {
+          db["col" + i].insert({"value": j});
+        }
+      }
+    },
+
+    tearDownAll: function () {
+      for (let i = 0; i < 3; i++) {
+        db._drop("col" + i);
+      }
+      db._useDatabase(prevDB);
+      db._dropDatabase(database);
+    },
+
+    testCalcRebalanceStopServer: function () {
+
+      let dbServers = instanceManager.arangods.filter(arangod => arangod.instanceRole === "dbserver");
+      try {
+        for (let i = 0; i < dbServers.length; ++i) {
```

Comment:
are we expecting `dbServers` to have a minimum length? what if `dbServers` is empty?